scripts/demo.py

```diff
@@ -1,23 +1,40 @@
 import argparse
 
 import cv2 as cv
+import numpy as np
 import glog as log
 from openvino.inference_engine import IECore
 
-from torchdet3d.utils import draw_kp, Regressor, Detector, OBJECTRON_CLASSES
+from torchdet3d.utils import draw_kp, Regressor, Detector, OBJECTRON_CLASSES, CameraTracker
 
+class SctConfig:
```

Comment:
A dict wrapped into a class looks strange, why not to use dataclass here? It can be converted to a dict by calling `dataclasses.asdict`.